### PR TITLE
feat(FRNet): apply softmax & fix dataset path

### PR DESCRIPTION
**Pull request by amadeuszsz**
## Summary

Apply softmax and deploy FRNet model with probabilities output as we do in PTv3.

Additionally fix the dataset path for test in deployment script.

### CODE REVIEW

File: projects/FRNet/deploy/trt_model.py [hunk 1]
```diff
@@ -151,13 +151,13 @@ def _run_engine(self, tensors: Dict[str, Dict]) -> None:
...
     def inference(self, batch_inputs_dict: dict) -> npt.NDArray[np.float32]:
-        """Run TensorRT inference, returns logits (N, num_classes)."""
+        """Run TensorRT inference, returns probabilities (N, num_classes)."""
         shapes_dict = {
             "points": batch_inputs_dict["points"].shape,
             "coors": batch_inputs_dict["coors"].shape,
             "voxel_coors": batch_inputs_dict["voxel_coors"].shape,
             "inverse_map": batch_inputs_dict["inverse_map"].shape,
-            "seg_logit": (batch_inputs_dict["points"].shape[0], self._deploy_cfg.num_classes),
+            "pred_probs": (batch_inputs_dict["points"].shape[0], self._deploy_cfg.num_classes),
```
[the remark this answers is not in the document]
Comment (amadeuszsz) [reply projects/FRNet/deploy/trt_model.py]: Nope, that's good.

File: projects/FRNet/configs/deploy/nuscenes/frnet_tensorrt_dynamic.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
     coors=dict(min_shape=[5000, 3], opt_shape=[30000, 3], max_shape=[60000, 3]),
     voxel_coors=dict(min_shape=[5000, 3], opt_shape=[30000, 3], max_shape=[60000, 3]),
     inverse_map=dict(min_shape=[5000], opt_shape=[30000], max_shape=[60000]),
-    seg_logit=dict(min_shape=[5000, num_classes], opt_shape=[30000, num_classes], max_shape=[60000, num_classes]),
+    pred_probs=dict(min_shape=[5000, num_classes], opt_shape=[30000, num_classes], max_shape=[60000, num_classes]),
```
[the remark this answers is not in the document]
Comment (amadeuszsz) [reply projects/FRNet/configs/deploy/nuscenes/frnet_tensorrt_dynamic.py]: https://github.com/tier4/AWML/pull/199/commits/a01a9303217e8ef40e8b571168c0b52b38b9499b

**Review by KSeangTan — APPROVED** [review]
LGTM overall, please resolve the comments and merge afterwards
